docs/tenancy/1.x/tutorial-non-tenant.md

```diff
@@ -0,0 +1,161 @@
+
+`app\Listeners\ConfigureTenantRoutes.php`
+```php
+namespace App\Listeners;
+
+use Tenancy\Affects\Routes\Events\ConfigureRoutes;
+
+class TenantRoutes 
+{
+    public function handle(ConfigureRoutes $event) 
+    {
+        if($event->event->tenant)
+        {
+            $event
+                ->flush()
+                ->fromFile(
+                    ['middleware' => ['web']],
+                    base_path('/routes/tenant.php')
+                );
+        }
+    }
+}
+```
+
+#### Registering the listener
+Now update your `app\Providers\EventServiceProvider.php` to include your new listener.
+
+```php
+  protected $listen = [
+
+    ...
+
+    Tenancy\Affects\Routes\Events\ConfigureRoutes::class => [
+      App\Listeners\ConfigureTenantRoutes::class,
+    ]
+    
+    ...
+  ];
+```
+
+## Status
+
+At this point you application is configured to load any tenant routes when a tenant is identified. When a tenant is not identified, the routes located in `routes/web.php` will be used instead.
+
+## Tips and Reminders
+### Namespaceing
+As with any Laravel application; if you are storing controllers in a different location than in `app\Http\Controllers` you will need to use the namespace option within your routes.
+
+`routes\tenant.php`
+```php
+Route::namespace('Tenant')->group(function () {
+    // Controllers Within The "App\Http\Controllers\Tenant" Namespace
+});
+```
+
+### Sub-site by domains
+When using subdomains to identify your non-tenant sub-sites you will need to use the route option within your routes file.
+`routes\web.php`
+```php
+Route::domain('admin.myapp.com')->group(function () {
+    Route::get('/', function () {
+        // This is the root route for the admin site
+    });
+});
+
+Route::domain('www.myapp.com')->group(function () {
+    Route::get('/', function () {
+        // This is the root route for the public site
+    });
+});
+
+Route::get('/shared', function() {
+  //This is a route that is shared with both the admin site and the public site
+})
+```
+
+### Accessing `OnTenant` models from a "sub-site"
+Before a model that uses the `OnTenant` trait can be used a tenant needs to be identified. Because of this it is best practice to avoid using tenant models on a non-tenant site.
+
+In events where this access is still needed, a tenant will first need to be identified within your code.
+
+The following example assumes you are trying to create an admin view that lists all products a specific tenant has in their store.
+
+`app\Http\Controllers\TenantProducts.php`
+```php
+public function index(Request $request)
+{
+  // We currently do not have a tenant identified as this is the "admin sub-site"
+
+  // Get our tenant (A Customer)
+  $customer = App\Models\Customer::firstOrFail($request->get('tenant_id'));
+  // Tell tenancy, that we want to access the above tenant
+  Tenancy::setTenant($customer)
```

Comment:
Would recommend using the `ConnectionResolver` in order to set the connection, rather than completely setting the tenant as per performance, security etc.